train_management/urls.py

```diff
@@ -24,6 +24,8 @@
     path('dayplanning/add/', views.DayPlanningCreateView.as_view(), name='day-planning-create'),
     path('dayplanning/<int:pk>/delete/', views.DayPlanningDeleteView.as_view(), name='day-planning-delete'),
 
+    path('dayplanning/<int:pk>/briefing', views.briefing_pdf, name='briefing-pdf'),
```

Comment:
Not sure, but is there a reason all the other routes have an ending slash and this one doesn't?